tanjun/commands.py

```diff
@@ -1786,6 +1786,7 @@ def add_mentionable_option(
     async def _process_args(self, ctx: abc.SlashContext, /) -> collections.Mapping[str, typing.Any]:
         keyword_args: dict[str, typing.Union[int, float, str, hikari.User, hikari.Role, hikari.InteractionChannel]] = {}
         for tracked_option in self._tracked_options.values():
+            member: typing.Optional[hikari.InteractionMember]
```

Comment:
Previously, two branches (`hikari.OptionType.USER` and `hikari.OptionalType.MENTIONABLE`) would set this type and that would annoy mypy.

Reply:
you can just keep the type hint on the first declaration instead of moving it up here